style.css

```diff
@@ -0,0 +1,69 @@
+body{
+    margin: 0;
+    background-repeat:no-repeat;
+    background-size: cover;
+    background-image: linear-gradient(rgba(58, 58, 158, 0.8), rgb(186, 113, 187)), url("https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg");
+    height: 230vh;
+    opacity: 0.8;
+    padding: 0px;
+    color: whitesmoke;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+header{
+    color: #fff;
+    text-align: center;
+}
+#container{
```

Comment:
This CSS  class selector is not properly used, kindly use the proper CSS syntax .

Reply:
- [ ] You forgot to explain how the student used the selector incorrectly and why he should use it the way you are requesting. In cases like this, you can even include an example for the student so that they know how the selector should be used. 👍🏾